components/TabList/tab-list.tsx

```diff
@@ -37,13 +37,13 @@ const TabList = ({ tabList, pageId, selectedTab }: TabListProps) => {
           role="tab"
           aria-selected={selectedTab === tab.name.toLowerCase() ? "true" : "false"}
           data-state={selectedTab === tab.name.toLowerCase() ? "active" : "inactive"}
-          tabIndex={-1}
           key={index}
-          className={`tool-list-item border-b-2 transition-all ease-in-out ${
+          className={clsx(
```

Comment:
After reviewing the pull request, this really is just links in a navigation element that look like tabs, but they really are just. links.

The `role="tablist"` can be removed from the `<nav />` and the `role="tab"` on the `<div />` can be removed.